docs/administrators/auth.rst

```diff
@@ -242,7 +242,7 @@ issued by keycloak.
    Show the correct configuration parameters in JSON format.
 
 Add a keycloak configuration parameters to the dashboard section of the server configuration file.
-(/etc/inmanta/server.cfg in most installs.) This section should already contain enabled=true and the
+(/etc/inmanta/inmanta.d/<custom-config>.cfg in most installs.) This section should already contain enabled=true and the
```

Comment:
add code block with the example section in it?

Reply:
and perhaps name the file /etc/inmanta/inmanta.d/dashboard.cfg

Reply:
I didn't add the requested code block. It was already present in the file.